spec/vcr_configurer.rb

```diff
@@ -21,6 +21,16 @@ def self.filter_body_field(vcr_config, part, field)
         end
       end
 
+      def self.filter_sso(vcr_config)
+        part = 'request'
+        placeholder = name_to_placeholder('PGP')
+        vcr_config.filter_sensitive_data(placeholder) do |interaction|
+          login = interaction[part].uri.end_with? '/gdc/account/customerlogin'
+          require 'pry';binding.pry if login
```

Comment:
warning: Style/Semicolon: Do not use semicolons to terminate expressions. (rubocop)